styles/_header.scss

```diff
@@ -68,52 +69,65 @@
 
 .nav-wrapper {
   a {
```

Comment:
can we target a class and not the tag?

Reply:
Sure. There will be parts of this codebase that I likely won't refactor to do that. Just warning you!